evm/db/backends/memory.py

```diff
@@ -6,8 +6,8 @@
 class MemoryDB(BaseDB):
     kv_store = None
 
-    def __init__(self):
-        self.kv_store = {}
+    def __init__(self, kv_store=None):
+        self.kv_store = {} if kv_store is None else kv_store
```

Comment:
*nitpick*

Inline turnary statements aren't great readability.  Do you mind expanding this to a fill `if/else` across 4 lines.